student/pom.xml

```diff
@@ -259,14 +257,12 @@
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-jdbc</artifactId>
-			<version>${spring.version}</version>
 		</dependency>
 
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-test</artifactId>
-			<version>${spring.version}</version>
-			<scope>test</scope>
+			<scope>compile</scope>
```

Comment:
Can you add a note in the XML that explains why this isn't a test scope?  Something like `Classes in spring-test are currently used within non test code`